src/wire/ip.rs

```diff
@@ -126,6 +127,55 @@ impl fmt::Display for Address {
     }
 }
 
+/// A CIDR containing a prefix of an IP subnet.
```

Comment:
I've been uncomfortable calling this data structure "CIDR" for a while but I just realized what the proper name for it would be: `IpSubnet`! That makes much more sense. I'm sorry for having you change everything.

Reply:
Subnets have their own semantics, for example, 192.168.1.0/24 is exactly the same subnet as 192.168.1.255/24, this makes accepting a slice of subnets as interface addresses somewhat confusing. I've changed the documentation to better reflect the fact that CIRD != subnet, but I'd prefer to keep `Cidr` as the name.